projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/container/multi-step-checkout.component.html

```diff
@@ -106,7 +106,11 @@ <h1 class="cx-page__title d-none d-lg-block d-xl-block">
                 />
                 <span class="form-check-label">
                   I confirm that I have read and agree with the
-                  <a href="#">Terms & Conditions</a>
+                  <a
+                    href="/terms-and-conditions"
```

Comment:
I think we can use `routerLink` instead basic href. What you think?

Reply:
Sure, good idea, will refactor